e2e/packages/redirections/src/index.tsx

```diff
@@ -169,6 +169,10 @@ const redirections: Redirections = {
           state.source.redirections = query.redirections;
         }
 
+        if (state.frontity.options.url) {
+          state.frontity.url = state.frontity.options.url;
```

Comment:
nice trick. I worry sometimes that the Frontity Options can be an attack vector if someone (god forbid) `evals` the contents of a frontity option or passes it as a parameter to a `fetch()` call etc.

I ll ping DevRel to explicitly warn about this in the docs.

Reply:
Good idea :+1: